From: Dmitry Kazakov <dimula73@gmail.com>
Date: Wed, 29 Apr 2015 11:04:40 +0000
Subject: Fast and dirty fix for loading PSD files
X-Git-Url: http://quickgit.kde.org/?p=calligra.git&a=commitdiff&h=223e938ca5c2bd4724b4e2d1e59f72971e2977bf
---
Fast and dirty fix for loading PSD files

PsdAdditionalLayerInfoBlock must know about the layers block size and not try to read
it infinitely. So now just not signal errors, when it is finished.
---


--- a/krita/plugins/formats/psd/psd_additional_layer_info_block.cpp
+++ b/krita/plugins/formats/psd/psd_additional_layer_info_block.cpp
@@ -41,7 +41,7 @@
         b = io->peek(4);
         if (b.size() != 4 || QString(b) != "8BIM") {
             error = "No 8BIM marker for additional layer info block";
-            return false;
+            return true;
         }
         else {
             io->seek(io->pos() + 4); // skip the 8BIM header we peeked ahead for

